database/index.js

```diff
@@ -0,0 +1,69 @@
+const {Sequelize} = require('sequelize');
+const orm = new Sequelize('review', 'student', 'student', {dialect: 'mysql'});
```

Comment:
Consider putting sensitive information such as your database login/pass in a git ignore.

Reply:
I am doing this now.

Reply:
Awesome!